.github/workflows/push.yaml

```diff
@@ -59,7 +60,7 @@ jobs:
           INSTALL_K3S_VERSION: ${{ matrix.K3S_VERSION }}
           K3S_KUBECONFIG_MODE: "644"
         run: |
-          /usr/local/bin/k3s-uninstall.sh        
+          /usr/local/bin/k3s-uninstall.sh || true
```

Comment:
Will this not cause troubles when K3S is not installed for some reason? Why did you remove this?

Reply:
It isn't removed, it's added

Reply:
Oh perfect then!